keiko-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlQueueProperties.kt

```diff
@@ -0,0 +1,49 @@
+package com.netflix.spinnaker.config
+
+import com.netflix.spinnaker.kork.sql.config.SqlRetryProperties
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.validation.annotation.Validated
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Positive
+
+@ConfigurationProperties("keiko.queue.sql")
+@Validated
+class SqlQueueProperties {
+  /**
+   * Enables use of the SqlQueue implementation, disabling RedisQueue.
+   */
+  var enabled: Boolean = false
+  /**
+   * [queueName] namespaces the database tables so that multiple keiko queues can be collocated
+   * on the same database. When using a globally writeable data store with queue processors in
+   * multiple regions, it may be desirable to use the region as the [queueName]. Must match
+   * regexp "\w+".
+   */
+  @Pattern(regexp = "^\\w+$")
+  var queueName: String = "default"
+  /**
+   * [deadLetterQueueName] defaults to [queueName] but can be set independently so that multiple
+   * queues on the same database can potentially share a DLQ. Must match regexp "\w+".
+   */
+  @Pattern(regexp = "^\\w+$")
+  var deadLetterQueueName: String = queueName // Can be shared across queues on the same master
+  /**
+   * The length of time in seconds that a handler has to complete processing a message, and
+   * acknowledge that processing has been completed. Messages that are not completed within
+   * [ackTimeoutSeconds] are returned to the queue, up-to [Queue.maxRetries] times.
+   */
+  @Positive(message = "ackTimeoutSeconds must be a positive integer")
+  var ackTimeoutSeconds: Long = 120
```

Comment:
What do you think about changing this config to a `Duration` type instead?

```
var ackTimeout: Duration = Duration.ofMinutes(2)

// And in config...

keiko.queue.sql.ackTimeout: 90s
```

Spring will handle conversions correctly. You'd also get the benefit of not requiring a validation of if the value is positive or not, as you cannot have below-zero durations (I don't think?)